Orange/widgets/visualize/owscatterplot.py

```diff
@@ -380,8 +380,14 @@ def update_graph(self, attributes=None, reset_view=True, **_):
         self.graph.zoomStack = []
         if attributes and len(attributes) == 2:
             self.attr_x, self.attr_y = attributes
+        X = BasicStats(self.data[:, self.attr_x], self.attr_x)
+        Y = BasicStats(self.data[:, self.attr_y], self.attr_y)
```

Comment:
I have 1 idea:
Create an array of size n(n=no of attributes) where arr[i] = 1 if ith attribute contains NaN values else arr[i] = 0.
But this will take `O(n*m)` m = no of tuples.
This will definitely be optimal solution. What do you think?